examples/components/MNIST/aggregatemodelweights/conda.yaml

```diff
@@ -0,0 +1,9 @@
+name: mnist_agg_conda_env
+channels:
+  - defaults
```

Comment:
for a proper install of torch through conda, you will want to add pytorch as a channel + install torch and torchvision with conda and not pip.

ex:

```
channels:
  - defaults
  - pytorch
dependencies:
  - pytorch=1.12.1
  - torchvision=0.13.1
  - cudatoolkit=11.3
```

Reply:
Gottu! Did the required changes.